pkgs/data/fonts/kacst/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "kacst";
+  version = "2.01";
+
+  src = fetchurl {
+    url = "mirror://debian/pool/main/f/fonts-kacst/fonts-kacst_${version}+mry.orig.tar.bz2";
+    sha256 = "6f2899ce9622314ea426cf8d48849f1cf17302726d73cb683486d5fdf2a23338";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/kacst
+    cp kacst/*.ttf $out/share/fonts/kacst
+  '';
```

Comment:
This could be done in postFetch of fetchurl to remove the extra derivation.

Reply:
@SuperSandro2000 thanks for your feedback, I'll change it.